bundles/org.openhab.binding.radiothermostat/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -157,6 +158,14 @@
 		<state min="0" max="2" pattern="%d"/>
 	</channel-type>
 
+	<channel-type id="remote_temp" advanced="true">
+		<item-type>Number</item-type>
```

Comment:
Please apply openHab's [Units of Measurement](https://www.openhab.org/docs/concepts/units-of-measurement.html) framework wherever applicable. This allows user interfaces greater flexibility when it comes to rendering data from your binding.
```suggestion
		<item-type>Number:Temperature</item-type>
```
Please adjust the pattern appropriately.

Reply:
done